data/api/client-server/key_backup.yaml

```
@@ -820,7 +820,7 @@ paths:
       summary: Retrieve the keys from the backup for a given room
       description: |-
         Retrieve the keys from the backup for a given room
-      operationId: getRoomKeysKeyRoomId
+      operationId: getRoomKeysKey
```

```suggestion
      operationId: getRoomKeys
```
I guess, the original author used the `Key` infix where `By` is used in other places of the spec (see, e.g. `directory.yaml` or `rooms.yaml`).